data/engine-cli/docker_tag.yaml

```diff
@@ -13,7 +13,7 @@ long: |-
     canonical reference for Docker's public registry.
     - `PORT_NUMBER`: If a hostname is present, it may optionally be followed by a
     registry port number in the format `:8080`.
-    - `PATH`: The path consists consists of slash-separated components. Each
```

Comment:
The yaml files in this repository are generated from the upstream docker CLI repository, and should not be modified manually (otherwise those changes get reverted the next time the generated files are updated).

The source for this part of the yaml can be found in the docker/cli repository; https://github.com/docker/cli/blob/master/docs/reference/commandline/tag.md

Perhaps you're interested in contributing there? If you do, make sure to read the CONTRIBUTING.md in that repo, as that repository requires commits to have a DCO sign-off line in the commit message